src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
```

Comment:
Ensure that the font-family list ends with a generic family; in this case, 'sans-serif' is already specified, so that's good. Just make sure to follow this practice consistently as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).